packages/dht/src/connection/websocket/BrowserWebsocketClientConnection.ts

```diff
@@ -0,0 +1,44 @@
+import { Logger } from '@streamr/utils'
+import { ICloseEvent, IMessageEvent, w3cwebsocket as Websocket } from 'websocket'
+import { AbstractWebsocketClientConnection } from './AbstractWebsocketClientConnection'
+
+const logger = new Logger(module)
+
+const BINARY_TYPE = 'arraybuffer'
+
+export class WebsocketClientConnection extends AbstractWebsocketClientConnection {
+
+    protected socket?: Websocket 
+
+    // TODO explicit default value for "selfSigned" or make it required
+    public connect(address: string, selfSigned?: boolean): void {
+        if (!this.destroyed) {
+            this.socket = new Websocket(address, undefined, undefined, undefined, { rejectUnauthorized: !selfSigned })
+            this.socket.binaryType = BINARY_TYPE
+            this.socket.onerror = (error: Error) => this.onError(error)
+            this.socket.onopen = () => this.onOpen()
+            this.socket.onclose = (event: ICloseEvent) => this.onClose(event.code, event.reason)
+            this.socket.onmessage = (message: IMessageEvent) => {
+                if (!this.destroyed) {
+                    if (typeof message.data === 'string') {
+                        logger.debug('Received string: \'' + message.data + '\'')
```

Comment:
This in unsupported feature? Could mention that in the log message (also in `WebsocketClientConnection`)